packages/backend/src/lib.rs

```diff
@@ -101,11 +104,37 @@ impl TryFrom<&str> for EventType {
     }
 }
 
+fn init_sentry() -> Option<sentry::ClientInitGuard> {
+    match env::var("SENTRY_DSN") {
+        Ok(sentry_dsn) => Some(sentry::init((
+            sentry_dsn,
+            sentry::ClientOptions {
+                release: sentry::release_name!(),
+                ..Default::default()
+            },
+        ))),
+        Err(_) => None,
+    }
+}
+
 pub async fn init<A: Into<String>>(
     actor_id: A,
     storage_path: Option<impl AsRef<Path>>,
+    send_diagnostics: Option<bool>,
     message_receiver: Arc<Mutex<Sender<String>>>,
 ) {
+    let send_diagnostics = match send_diagnostics {
+        Some(arg) => arg,
+        None => false,
+    };
```

Comment:
There's a warning from `cargo clippy` suggesting that this can be written more concisely:

```
warning: this pattern reimplements `Option::unwrap_or`
   --> src/lib.rs:126:28
    |
126 |       let send_diagnostics = match send_diagnostics {
    |  ____________________________^
127 | |         Some(arg) => arg,
128 | |         None => false,
129 | |     };
    | |_____^ help: replace with: `send_diagnostics.unwrap_or(false)`
    |
    = note: `#[warn(clippy::manual_unwrap_or)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#manual_unwrap_or
```

Reply:
Thank you for the suggestion, Rajiv! I've added it to the PR :slightly_smiling_face: